setup.py

```diff
@@ -34,7 +34,7 @@
 
 
 setup(name='odoo-client-lib',
```

Comment:
[Modernize the project?](https://packaging.python.org/en/latest/guides/modernize-setup-py-project/)

Reply:
done in https://github.com/odoo/odoo-client-lib/pull/5/commits/7e7ca04c9e8ce45cce22433daf91f00583700e6c